rpi-update

```diff
@@ -115,10 +115,39 @@ function update_sdk {
 }
 
 function show_notice {
-	local NOTICE=`curl -Lfs https://raw.githubusercontent.com/hexxeh/rpi-firmware/${FW_REV}/NOTICE.md`
+	local FULL_NOTICE=`curl -Lfs https://raw.githubusercontent.com/hexxeh/rpi-firmware/${FW_REV}/NOTICE.md`
+	local NOTICE=$(echo "$FULL_NOTICE" | tail -n+2)
 	if [ -z "$NOTICE" ]; then
```

Comment:
I wonder if this should be checking $FULL_NOTICE instead of $NOTICE ?
I think as it is currently, it wouldn't display the NOTICE if it was only one line long?

Reply:
In fact, I think this line should be `if [ -z "$FULL_NOTICE" ]; then` and the `local NOTICE=$(echo "$FULL_NOTICE" | tail -n+2)` can probably be moved down to line 138 or so.